Tasks/AzureFileCopy/task.json

```diff
@@ -141,6 +141,15 @@
       "helpMarkDown": "Additional AzCopy.exe arguments that will be applied when uploading to blob or uploading to VM like, /NC:10."
     },
     {
+      "name": "enableRemoteDeployment",
```

Comment:
Can we think of better name. Remote deployment made sense in other tasks. But it would be very hard for someone using copy task to relate and enable this option.